jboss/container/wildfly/launch/json-logging/added/launch/json_logging.sh

```diff
@@ -9,8 +9,10 @@ function configure_json_logging() {
   getConfigurationMode "##CONSOLE-FORMATTER##" "configureMode"
   if [ "${configureMode}" = "xml" ]; then
     configureByMarkers
+    configurePatternFormatterByMarkers
   elif [ "${configureMode}" = "cli" ]; then
     configureByCLI
+    configurePatternFormatterByCLI
```

Comment:
This call will not have any effect. the generated CLI operations needs to be added to the CLI script file.
Should be: configurePatternFormatterByCLI >> ${CLI_SCRIPT_FILE}